captcha/locale/ar/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,24 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Django-recapcha\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-12-26 08:32-0500\n"
+"PO-Revision-Date: 2014-09-15 16:47-0300\n"
+"Last-Translator: Rafael Ponieman <rafadev@gmail.com>\n"
+"Language-Team: \n"
+"Language: Spanish\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
```

Comment:
**Question:** are you sure these plural forms are correct for Arabic? [This source suggests there exist 6 plural forms in Arabic and gives a different formula](https://wiki.arabeyes.org/Plural_Forms).